### ISSUE TITLE
v4.2.3 osx (darwin) release artifact is missing 

### ISSUE THREAD

**Issue by lalyos** [rqlite/rqlite#396]
```
$ curl -I  -L https://github.com/rqlite/rqlite/releases/download/v4.2.3/rqlite-v4.2.3-darwin-amd64.tar.gz
HTTP/1.1 404 Not Found
```

**Comment by otoolep** [comment]
You're right. :-( I lost access to the Mac and didn't get a chance to build it yet. Hope to do so within days. In the meantime you can find the build instructions here:

https://github.com/rqlite/rqlite/blob/master/CONTRIBUTING.md#building-rqlite

4.2.2 is very similar to 4.2.3, and I did upload 4.2.2 for OSX.

**Comment by otoolep** [comment]
I have actually released a new version, v4.3.0, and it is available for OSX.
